services/Embedder.py

```diff
@@ -0,0 +1,62 @@
+from abc import ABC, abstractmethod
+from discord import Embed
+from util.Utility import Colors
```

Comment:
No longer exists after deletion of `util.Utility`.
```suggestion
from discord import Colour
```

Reply:
Had to fiddle with this a bit - apparently there was a difference between `from discord import Colour` and `from discord,colour import Colour`.  Resolved now.

Additionally, `from discord,colour import Color` works as well - this is what I ran with because because